packages/DataHeader/stories/DataHeader.stories.mdx

```diff
@@ -0,0 +1,158 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import { getStoryName } from "storybook/storyTree";
+import Readme from "../README.md";
+import { Basic, WithRenderActions, WithTypes, WithBackgroundColor } from "./DataHeader.examples";
+import Toast from "@paprika/toast";
+
+export const params = {
+  options: {
+    isToolshown: true,
+    showPanel: false,
+  },
+};
+
+<Meta title={`${getStoryName("DataHeader")}/Docs`} parameters={params} />
+
+# README
+
+<Description markdown={Readme} />
```

Comment:
Without the import above, this doesn't render (but also doesn't produce an error, I guess because it's MDX).

Reply:
do you mean in the linter? we are not lingering mdx as you commenting